### ISSUE TITLE
pulseaudio does not compile on Ubuntu 16.04 LTS

### ISSUE THREAD

**Issue by retokromer** [Linuxbrew/homebrew-core#1174]
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed as many issues as possible and retried your prior step?
### Bug reports:

`brew install pulseaudio` gives:

```
==> Downloading https://www.freedesktop.org/software/pulseaudio/releases/pulseau
Already downloaded: /home/why/.cache/Homebrew/pulseaudio-9.0.tar.xz
==> Downloading https://raw.githubusercontent.com/Homebrew/formula-patches/15fa4
Already downloaded: /home/why/.cache/Homebrew/pulseaudio--patch-d3a2180600a4fbea538949b6c4e9e70fe7997495663334e50db96d18bfb1da5f.patch
==> Patching
==> Applying i386.patch
patching file src/pulsecore/svolume_mmx.c
patching file src/pulsecore/svolume_sse.c
==> ./configure --disable-silent-rules --prefix=/home/why/.linuxbrew/Cellar/puls
==> make install
Last 15 lines from /home/why/.cache/Homebrew/Logs/pulseaudio/02.make:
/usr/bin/gcc-5 -DHAVE_CONFIG_H -I. -I..  -I../src -I../src/modules -I../src/modules -DPA_ALSA_PATHS_DIR=\"\" -DPA_ALSA_PROFILE_SETS_DIR=\"\" -DPA_SRCDIR=\"/tmp/pulseaudio-20161030-10635-1yml191/pulseaudio-9.0/src\" -DPA_BUILDDIR=\"/tmp/pulseaudio-20161030-10635-1yml191/pulseaudio-9.0/src\" -DPULSE_LOCALEDIR=\"/home/why/.linuxbrew/Cellar/pulseaudio/9.0/share/locale\" -isystem/home/why/.linuxbrew/include -DFASTPATH -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 -std=gnu11 -pthread  -Os -w -pipe -march=native -Wall -W -Wextra -Wno-long-long -Wno-overlength-strings -Wunsafe-loop-optimizations -Wundef -Wformat=2 -Wlogical-op -Wsign-compare -Wformat-security -Wmissing-include-dirs -Wformat-nonliteral -Wold-style-definition -Wpointer-arith -Winit-self -Wdeclaration-after-statement -Wfloat-equal -Wmissing-prototypes -Wstrict-prototypes -Wredundant-decls -Wmissing-declarations -Wmissing-noreturn -Wshadow -Wendif-labels -Wcast-align -Wstrict-aliasing -Wwrite-strings -Wno-unused-parameter -ffast-math -fno-common -fdiagnostics-show-option -fdiagnostics-color=auto -c -o utils/pasuspender-pasuspender.o `test -f 'utils/pasuspender.c' || echo './'`utils/pasuspender.c
/usr/bin/gcc-5 -DHAVE_CONFIG_H -I. -I..  -I../src -I../src/modules -I../src/modules -DPA_ALSA_PATHS_DIR=\"\" -DPA_ALSA_PROFILE_SETS_DIR=\"\" -DPA_SRCDIR=\"/tmp/pulseaudio-20161030-10635-1yml191/pulseaudio-9.0/src\" -DPA_BUILDDIR=\"/tmp/pulseaudio-20161030-10635-1yml191/pulseaudio-9.0/src\" -DPULSE_LOCALEDIR=\"/home/why/.linuxbrew/Cellar/pulseaudio/9.0/share/locale\" -isystem/home/why/.linuxbrew/include -DFASTPATH -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 -std=gnu11 -pthread  -Os -w -pipe -march=native -Wall -W -Wextra -Wno-long-long -Wno-overlength-strings -Wunsafe-loop-optimizations -Wundef -Wformat=2 -Wlogical-op -Wsign-compare -Wformat-security -Wmissing-include-dirs -Wformat-nonliteral -Wold-style-definition -Wpointer-arith -Winit-self -Wdeclaration-after-statement -Wfloat-equal -Wmissing-prototypes -Wstrict-prototypes -Wredundant-decls -Wmissing-declarations -Wmissing-noreturn -Wshadow -Wendif-labels -Wcast-align -Wstrict-aliasing -Wwrite-strings -Wno-unused-parameter -ffast-math -fno-common -fdiagnostics-show-option -fdiagnostics-color=auto -c -o utils/pacmd-pacmd.o `test -f 'utils/pacmd.c' || echo './'`utils/pacmd.c
/usr/bin/gcc-5 -DHAVE_CONFIG_H -I. -I..  -I../src -I../src/modules -I../src/modules -DPA_ALSA_PATHS_DIR=\"\" -DPA_ALSA_PROFILE_SETS_DIR=\"\" -DPA_SRCDIR=\"/tmp/pulseaudio-20161030-10635-1yml191/pulseaudio-9.0/src\" -DPA_BUILDDIR=\"/tmp/pulseaudio-20161030-10635-1yml191/pulseaudio-9.0/src\" -DPULSE_LOCALEDIR=\"/home/why/.linuxbrew/Cellar/pulseaudio/9.0/share/locale\" -isystem/home/why/.linuxbrew/include -DFASTPATH -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 -std=gnu11 -pthread  -I/home/why/.linuxbrew/Cellar/libx11/1.6.3/include -I/home/why/.linuxbrew/Cellar/libsm/1.2.2/include -I/home/why/.linuxbrew/Cellar/libice/1.0.9/include -I/home/why/.linuxbrew/Cellar/libxtst/1.2.2/include -I/home/why/.linuxbrew/Cellar/libxi/1.7.6/include -I/home/why/.linuxbrew/Cellar/libxext/1.3.3/include -I/home/why/.linuxbrew/Cellar/libxfixes/5.0.1/include -I/home/why/.linuxbrew/Cellar/libx11/1.6.3/include -I/home/why/.linuxbrew/Cellar/libxcb/1.11.1/include -I/home/why/.linuxbrew/Cellar/libxau/1.0.8/include -I/home/why/.linuxbrew/Cellar/libxdmcp/1.1.2/include -I/home/why/.linuxbrew/Cellar/kbproto/1.0.7/include -I/home/why/.linuxbrew/Cellar/xproto/7.0.28/include -I/home/why/.linuxbrew/Cellar/fixesproto/5.0/include -I/home/why/.linuxbrew/Cellar/xextproto/7.3.0/include -I/home/why/.linuxbrew/Cellar/inputproto/2.3.1/include -I/home/why/.linuxbrew/Cellar/recordproto/1.14.2/include -Os -w -pipe -march=native -Wall -W -Wextra -Wno-long-long -Wno-overlength-strings -Wunsafe-loop-optimizations -Wundef -Wformat=2 -Wlogical-op -Wsign-compare -Wformat-security -Wmissing-include-dirs -Wformat-nonliteral -Wold-style-definition -Wpointer-arith -Winit-self -Wdeclaration-after-statement -Wfloat-equal -Wmissing-prototypes -Wstrict-prototypes -Wredundant-decls -Wmissing-declarations -Wmissing-noreturn -Wshadow -Wendif-labels -Wcast-align -Wstrict-aliasing -Wwrite-strings -Wno-unused-parameter -ffast-math -fno-common -fdiagnostics-show-option -fdiagnostics-color=auto -c -o utils/pax11publish-pax11publish.o `test -f 'utils/pax11publish.c' || echo './'`utils/pax11publish.c
/bin/sed -e "s|@pkglibdir[@]|/home/why/.linuxbrew/Cellar/pulseaudio/9.0/lib/pulseaudio|g" utils/padsp.in > padsp
make[3]: *** No rule to make target 'daemon/pulseaudio.desktop', needed by 'all-am'. Stop.
make[3]: *** Waiting for unfinished jobs....
make[3]: Leaving directory '/tmp/pulseaudio-20161030-10635-1yml191/pulseaudio-9.0/src'
Makefile:10988: recipe for target 'install' failed
make[2]: *** [install] Error 2
make[2]: Leaving directory '/tmp/pulseaudio-20161030-10635-1yml191/pulseaudio-9.0/src'
Makefile:807: recipe for target 'install-recursive' failed
make[1]: *** [install-recursive] Error 1
make[1]: Leaving directory '/tmp/pulseaudio-20161030-10635-1yml191/pulseaudio-9.0'
Makefile:1106: recipe for target 'install' failed
make: *** [install] Error 2

READ THIS: https://github.com/Linuxbrew/brew/blob/master/docs/Troubleshooting.md#troubleshooting
If reporting this issue please do so at (not Homebrew/brew):
  https://github.com/Linuxbrew/homebrew-core/issues
```

And `brew gist-logs pulseaudio` gives:

https://gist.github.com/48113901bcb95dd12c4019c2fdf20097

**Comment by sjackman** [comment]
Try
```sh
HOMEBREW_MAKE_JOBS=1 brew install pulseaudio
```

If possible, reinstall Linuxbrew in `/home/linuxbrew/.linuxbrew` to make use of the precompiled binary bottle.

You can alternatively try `brew install --force-bottle pulseaudio` with your current installation, but it may or may not work.

**Comment by retokromer** [comment]
Thank you!

`HOMEBREW_MAKE_JOBS=1 brew install pulseaudio` gives the same error.

`brew install --force-bottle pulseaudio` works fine.

I will try to reinstall Linuxbrew in `/home/linuxbrew/.linuxbrew`.

**Comment by jpauwels** [comment]
I've got exactly the same error on CentOS 7.2. `HOMEBREW_MAKE_JOBS=1` does not help either.

**Comment by sjackman** [comment]
If possible, reinstall Linuxbrew in `/home/linuxbrew/.linuxbrew` or try `brew install --force-bottle pulseaudio`

**Comment by jpauwels** [comment]
The bottled install seems to work for now, but I just wanted to let you know this is still an outstanding problem.

**Comment by sjackman** [comment]
Hi, Johan. This bug appears to be a known upstream issue: https://bugs.freedesktop.org/show_bug.cgi?id=54658
Duplicate of https://github.com/Homebrew/homebrew-core/issues/2408

**Comment by sjackman** [comment]
@rwhogg I just ran into this issue as well. Do you have a fix? I'm using the workaround of
```sh
brew install --force-bottle pulseaudio
```

**Comment by jpauwels** [comment]
@sjackman @rwhogg I've confined the error in #1347

**Comment by sjackman** [comment]
Fixed by Homebrew#7779.
